canvas.html

```diff
@@ -0,0 +1,249 @@
+            drawImages();
+            updateSelectedImageInfo();
+        } else if (event.key === 'ArrowUp') {
+            moveImageUp();
+            event.preventDefault();
+        } else if (event.key === 'ArrowDown') {
+            moveImageDown();
+            event.preventDefault();
+        }
+    }
+
+    function getStickerIdsFromUrl() {
+        const params = new URLSearchParams(window.location.search)
+        const slot0 = params.get("slot0")
+        const slot1 = params.get("slot1")
+        const slot2 = params.get("slot2")
+        const slot3 = params.get("slot3")
+        const slot4 = params.get("slot4")
+
+        return [slot0, slot1, slot2, slot3, slot4]
+    }
+
+    async function findStickerById(idToFind) {
+        const response = await fetch("https://cs-sticker.com/stickers.json")
+        const items = await response.json()
+        const item = items.find(item => item.id === idToFind);
+        return item ? item : null;
+    }
+
+    async function placeSticker() {
+        const stickerIds = getStickerIdsFromUrl();
+        const foundStickers = await Promise.all(stickerIds.map(id => findStickerById(id)));
+        console.log(foundStickers)
+        const validStickers = foundStickers.filter(sticker => sticker !== null);
+        console.log(validStickers)
+        validStickers.forEach((validSticker, index) => {
+            const imageName = validSticker.name;
+            const imagePath = validSticker.image;
+            // Calculate the offset for each sticker
+            const offSet = 50 * (index);
+            loadImage(imagePath, imageName, offSet);
+        });
+    }
+
+    function updateSelectedImageInfo() {
+        const selectedImage = images[selectedImageIndex];
+        const selectedImageInfoDiv = document.getElementById('selectedImageInfo');
+        if (selectedImage) {
+            selectedImageInfoDiv.innerHTML = `<p style="color: #fff; font-size: 2rem;">Selected on Canvas: </p><img src="${selectedImage.element.src}" class="selected-image"/>`;
+        } else {
+            selectedImageInfoDiv.innerText = 'No Image Selected';
+        }
+    }
+
+    function moveImageDown() {
+        if (selectedImageIndex !== null && selectedImageIndex > 0) {
+            const temp = images[selectedImageIndex];
+            images[selectedImageIndex] = images[selectedImageIndex - 1];
+            images[selectedImageIndex - 1] = temp;
+            selectedImageIndex--;
+            drawImages();
+        }
+    }
+
+    function moveImageUp() {
+        if (selectedImageIndex !== null && selectedImageIndex < images.length - 1) {
+            const temp = images[selectedImageIndex];
+            images[selectedImageIndex] = images[selectedImageIndex + 1];
+            images[selectedImageIndex + 1] = temp;
+            selectedImageIndex++;
+            drawImages();
+        }
+    }
+
+    document.getElementById('searchInput').addEventListener('input', function(event) {
```

Comment:
If you don't use the Searchbar and dropdown from my Site you can remove this function, it causes just an error if you leave it here